.github/workflows/yb-confluent-package.yml

```diff
@@ -0,0 +1,56 @@
+name: Create package for Confluent
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version of the connector to be packaged"
+        required: true
+        type: string
+      isSnapshotBuild:
+        description: "Snapshot build?"
```

Comment:
what is this?

Reply:
This is for the form which we will need to fill while launching this GitHub action.

Reply:
The yml file was copied from another file and the parameter `isSnapshotBuild` is not being used currently and is kept there for future use if needed.